src/Callback/Form.tsx

```diff
@@ -66,10 +68,19 @@ const CallbackForm: FunctionComponent = () => {
       if (response.kind === "success") {
         navigation.navigate(CallbackStackScreens.Success)
       } else {
+        Logger.addMetadata("requestCallbackError", {
+          errorMessage: response.message,
+        })
+        Logger.error(
+          `FailureToRequestCallback.${response.error}.${
+            response.message || defaultErrorMessage
```

Comment:
defaultErrorMessage is assigned to`" "` here, should we be using something different for the logger?